Doc/c-api/time.rst

```diff
@@ -0,0 +1,93 @@
+.. highlight:: c
+
+PyTime C API
+============
+
+.. versionadded:: 3.13
+
+The clock C API provides access to system clocks and time conversion functions.
+It is similar to the Python :mod:`time` module.
+
+For C API related to the :mod:`datetime` module, see :ref:`datetimeobjects`.
+
+
+Types
+-----
+
+.. c:type:: PyTime_t
+
+   A timestamp or duration.
+
+   The reference point for timestamps depends on the clock used. For example,
+   :c:func:`PyTime_Time` returns timestamps relative to the UNIX epoch.
+
+   ``PyTime_t`` is a signed integral type. It is possible to add and
+   subtract such values directly.
+   Keep in mind that C overflow rules do apply.
+
+   ``PyTime_t`` can represent time with nanosecond or better resolution.
+   The unit used is an internal implementation detail; use
+   :c:func:`PyTime_AsNanoseconds` or :c:func:`PyTime_AsSecondsDouble`
+   to convert to a given unit.
+
+   The supported range is at least around [-292.3 years; +292.3 years].
+   Using the Unix epoch (January 1st, 1970) as reference, the supported date
+   range is at least around [1677-09-21; 2262-04-11].
+   The exact limits are exposed as constants.
+
+   .. c:var:: PyTime_t PyTime_MIN
```

Comment:
This variable is declared *inside* PyTime_t, and so Sphinx fails to find it Doc/whatsnew/3.13.rst. Maybe remove the indentation?